lib/playground.dart

```diff
@@ -579,6 +582,22 @@ ${info['libraryName'] == null ? "" : "**Library:** ${apiLink == null ? info['lib
         messageSpan.classes.add('message');
         messageSpan.text = issue.message;
         e.children.add(messageSpan);
+        source.offset = issue.charStart;
+        dartServices.fix(source).timeout(serviceCallTimeout).then((FixesResponse fixResponse) {
+          if (fixResponse.fixes.isNotEmpty) {
```

Comment:
@devoncarew Lukechurch asked if I could add this. What this does is making sure the wrench is shown only if there is a quickfix available:

![image](https://cloud.githubusercontent.com/assets/1035299/7004264/1ed63680-dc69-11e4-8d6a-446c8fcc3151.png)